rdrf/rdrf/services/io/notifications/email_notification.py

```diff
@@ -258,6 +259,25 @@ def process_notification(reg_code=None, description=None, template_data={}):
             logger.warning("Email %s disabled" % note)
             has_disabled = True
             continue
-        send_result = process_given_notification(note, template_data)
+        send_result = process_given_notification(note, template_data, additional_recipients)
         sent_successfully = sent_successfully and send_result
     return sent_successfully, has_disabled
+
+
+def process_notification_with_default(reg_code=None, description=None, template_data={}, default_template=None, default_subject=None, default_recipient=[]):
+    notes = EmailNotification.objects.filter(registry__code=reg_code, description=description)
+
+    if notes:
+        return process_notification(reg_code, description, template_data, default_recipient)
```

Comment:
Need to check, but I think this will also send an activation email to the user's current email address, which is not the desired behaviour.

Reply:
No, it's all good as long as the person configuring the email notification leaves the recipient field blank.